res/tiles.json

```diff
@@ -4,55 +4,63 @@
     "name": "Default",
     "terminal": "?",
     "hardness": 1,
-    "texture": [0, 0]
+    "texture": [0, 0],
+    "isSolid": false
```

Comment:
Do we want default false? If stone misloads we don't want player to be falling infinitely.